examples/research_projects/realfill/train_realfill.py

```diff
@@ -450,10 +450,10 @@ def __init__(
 
         self.transform = transforms_v2.Compose(
             [
+                transforms_v2.ToImage(),
```

Comment:
I also put the Tensor conversion at the beginning *before* the resize, following the [perf recommendations](https://pytorch.org/vision/main/transforms.html#performance-considerations)

Reply:
nice!